dali/pipeline/operator/operator.h

```diff
@@ -80,17 +80,10 @@ class DLL_PUBLIC OperatorBase {
 
   DLL_PUBLIC virtual inline ~OperatorBase() {}
 
-  DLL_PUBLIC virtual bool Setup(std::vector<OutputDesc> &output_desc, const HostWorkspace &ws) {
-    DALI_FAIL("CPU execution is not implemented for this operator!");
+  DLL_PUBLIC virtual bool Setup(std::vector<OutputDesc> &output_desc, const Workspace &ws) {
+    DALI_FAIL("Not implemented!");
```

Comment:
If this is a message, that user will get when his operator is not available for the device he used (e.g. `fn.optical_flow(device='cpu')`), maybe it should be more verbose? Unless this error message won't ever get to the user (TBH I don't remember which was it).

Reply:
Now that I think of it, this function could now be a pure virtual function and an operator that doesn't implement it would just fail to compile... Like it always should have.